sklearn/metrics/cluster/supervised.py

```diff
@@ -25,14 +25,23 @@
 from ...utils.fixes import comb
 
 
-def comb2(n):
+def _comb2(n):
```

Comment:
Why do you change it? Will a docstring for the function solve the problem? I think we need to be careful when making things private.

Reply:
Yes a docstring would have solved the tests, I just thought putting it private was actually a better option because this function is only ever used locally

```
~/dev/scikit-learn(branch:master) » ag comb2                                                                                                                                       nico@cotier
sklearn/metrics/cluster/supervised.py
28:def comb2(n):
208:    sum_comb_c = sum(comb2(n_c) for n_c in np.ravel(contingency.sum(axis=1)))
209:    sum_comb_k = sum(comb2(n_k) for n_k in np.ravel(contingency.sum(axis=0)))
210:    sum_comb = sum(comb2(n_ij) for n_ij in contingency.data)
212:    prod_comb = (sum_comb_c * sum_comb_k) / comb2(n_samples)
```

We're having a similar discussion in #11407.